jobserver/templates/workspace_backend_files.html

```diff
@@ -54,7 +54,7 @@ <h1>Level 4 outputs from {{ workspace.name }}</h1>
       You can see these outputs because you have access to the Level 4 environment on the
       {{ backend.name }} backend and have been assigned the ProjectDeveloper role.
     </p>
-    <a class="btn btn btn-success" href="{{ release.get_download_url }}">
+    <a class="btn" disabled="true" aria-disabled="true">
```

Comment:
This template is only used on the L4 outputs page, so this button never be enabled, maybe we should remove and put some text in its place to explain how to release outputs?

Disabling `<a>`s is a pain, you need to set the `disabled` class instead: https://getbootstrap.com/docs/4.0/components/buttons/#disabled-state

Reply:
So, in general, in the simple cases I prefer disabled UI components rather than hidden ones, as it provides more information, and keeps the layout stable. A disabled button says to me "we've disabled the download on purpose", whereas the absence doesn't say anything either way.

But I don't want to introduce new UI paradigms here

Reply:
Oh, you want to keep it matching the other SPA pages, gotcha!  Yep, lets do that.  Maybe add a `<p>` above/below just to explain that you can't download from a L4 page, outputs must be released and link to the releasing docs?

Reply:
Have added a title which should show up as a tooltip?